canonical-web-and-design/ubuntu.com
# Move contributions js to build pipeline

## Done

- Moved inline script that handled contribution sliders to build pipeline

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/download/desktop/thank-you
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Use the sliders, see that the totals and the summary at the bottom update and that there are no console errors
